torchvision/prototype/transforms/_misc.py

```diff
@@ -131,6 +138,7 @@ def _transform(self, inpt: Any, params: Dict[str, Any]) -> Any:
 
 
 class ToDtype(Lambda):
+    # datumbox: if on Lambda we add a default behaviour for types, we might want to do the same here.
```

Comment:
cc @vfdev-5 / @pmeier

Reply:
The default should be different though, shouldn't it? `Lambda` should transform everything, while this should only apply to tensors.

Reply:
Good point, so something like:
```python
types = (torch.Tensor,)
```

Reply:
See https://github.com/pytorch/vision/pull/6492/commits/afa772ca3aa00f196ab10e9bdbe9b7f574716362

Reply:
Addressed by #6492